docs/contents.md

```diff
@@ -167,69 +215,83 @@ Following is the module schema of the **Tasks** module
+| Card Tiles                            | Shows a list of records in a card tile view                                                                                              |
+| Task Management                       | Allows users to manage tasks and get visibility into the current task board.                                                             |
+| Manage Datasets                       | Manage and Create datasets out of threat feeds                                                                                           |
+| Access Control                        | Change which teams or users have access to certain records                                                                               |
+| Feed Configuration Settings           | Configures feeds to be consumed by modules                                                                                               |
+| Threat Intel Management Configuration | Configures the Threat Intel Management solution pack                                                                                     |
+| TAXII Server Configuration            | Configures the TAXII Server for easy dissemination of information                                                                        |
+| Picklist as Phases                    | Displays the 'Report Status' picklist values as phases in a flow diagram format within the detail view of a threat intel report          |
+| FortiGuard IOC Search                 | Accepts indicators separated by a line break to get information from FortiGuard Labs                                                     |
+| Kill Chain Phases                     | Shows the IOC count present for different stages of cyber attack. It highlights the stage of the selected IOC                            |
+| Cyber Threat World Map                | A world map that displays top visitor origins to understand global exposure and potential threat spread                                  |
+| Summary grid                          | Creates a grid and populates it with formatted JSON data                                                                                 |
+| Speedometer                           | Displays the percentage value of selected parameter using a red dial gauge                                                               |
+| Categorical Insights                  | Creates division of data using different color codes                                                                                     |
+| Custom Tags                           | A series of tags that summarize associated CVE identifiers, named threats and exploits, and mapped outbreaks and exploitation techniques |
+| Card View                             | Dynamically displays JSON data as cards                                                                                                  |
```

Comment:
Update the "Card View" description - 
"Reads JSON data and dynamically generates a responsive card listing. Each card adapts to the available properties, offering a clean, structured preview of information."